### ISSUE TITLE
rvm regenerates wrappers unnecessarily

### ISSUE THREAD

**Issue by mwlang** [rvm/rvm#2725]
rvm is regenerating wrappers more frequently than it used to...almost every directory change seems to be triggering a regeneration of wrappers. for example:

```
Regenerating ruby-2.0.0-p451 wrappers.........
01:52:56:expenses >> cd ..
Regenerating ruby-2.0.0-p451 wrappers.........
01:53:08:rails4 >> cd expenses/
Regenerating ruby-2.0.0-p451 wrappers.........
01:53:13:expenses >> 
```

Since it takes about 3 seconds to generate the wrappers, its fairly annoying.  Can anything be done to either speed up or skip wrapper regeneration of the ruby version didn't actually change?

**Comment by barttenbrinke** [comment]
Not for me. It keeps writing to ~/.rvm/log with the text: gem gem-wrappers >=1.2.7 is already installed

**Comment by mpapis** [comment]
@barttenbrinke if you still have issues open a new ticket

**Comment by sonianand11** [comment]
With regards to this issue, I was facing same. I resolved it by removing respective hooks from `~/.rvm/hooks` directory. I have learned and understand more about hooks from here : https://rvm.io/workflow/hooks

Hope that helps!!!

**Comment by dimitko** [comment]
The only thing that worked for me basically deleting the `~/.rvm/hooks` directory (using user install, obviously).

I tried tons of stupid things which I won't even start to enumarate, but what @sonianand11 recommended was the only thing that made my Capistrano deployment work without constantly being spammed with "Regenerating ruby-X.X.X wrappers..." and 20 lines of "rotating" stick symbol below.

Also stopped the error of "/usr/bin/which: no bundle in ...". BTW, my project's gemset wasn't added in front of PATH and I had two RVM loaders both in `~/.bashrc` and `~/.profile` and I confirmed they are being loaded.
